### PR TITLE
build(ci): use travis scripts for ci instead of ci.sh

### PR DESCRIPTION
**Pull request by priyajeet**
1. Removes ci.sh script
2. Caches node_modules on travis
3. Parallelizes ci build tasks
4. Adds babel build
5. Removes externalized react webpack build

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @priyajeet has signed the CLA. Thanks for the pull request!

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,19 @@
 language: node_js
 node_js:
     - "8"
-script: ./build/ci.sh
```
Comment (jstoffan) [inline .travis.yml]: Can this script be deleted?

**Comment by boxcla** [comment]
# Webpack Bundle Size Github Bot

| Bundle        | Diff | Percentage Diff | Before | After |
| ------------- |:----:|:---------------:|:------:|:-----:|
| Total | 0 B | 0.00% | 0 B | 0 B |

Compared against previously merged PR #642

**Powered by [webpack-bundle-size-github-bot](https://github.com/wenboyu2/webpack-bundle-size-github-bot/)**

File: .travis.yml (hunk 1 above)
Comment (priyajeet) [reply .travis.yml]: done

**Pull request closed without merging**